.travis.yml

```diff
@@ -1,14 +1,16 @@
 language: node_js
 node_js:
+  - "8.12"
   - "8"
-  - "10.0"
+  - "10.3"
   - "10"
   - "12.0"
   - "12"
+  - "lts/*"
   - "node"
 
```

Comment:
In https://github.com/solid/solid-panes/pull/204/files you have just `install: ` - you want  `before_install` here?